ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -168,11 +173,63 @@ determine_old_or_new_fsl()
 have_hand_reclassification()
 {
 	local StudyFolder="${1}"
-	local Subject="${2}"
+	local Session="${2}"
 	local fMRIName="${3}"
 	local HighPass="${4}"
 
-	[ -e "${StudyFolder}/${Subject}/MNINonLinear/Results/${fMRIName}/${fMRIName}_hp${HighPass}.ica/HandNoise.txt" ]
+	[ -e "${StudyFolder}/${Session}/MNINonLinear/Results/${fMRIName}/${fMRIName}_hp${HighPass}.ica/HandNoise.txt" ]
+}
+
+# ------------------------------------------------------------------------------
+# Copy necessary files from cross-sectional to longitudinal target
+# ------------------------------------------------------------------------------
+function copy_to_longitudinal()
+{
+	local StudyFolder="$1" SessionCross="$2" SessionLong="$3" fMRINames="$4" ConcatName="$5" HighPass="$6"
```

Comment:
Can we list one local variable per line, for consistency with other code? (e.g., `have_hand_reclassification` just above)?
```suggestion
	local StudyFolder="${1}" 
	local SessionCross="${2}" 
	local SessionLong="${3}" 
	local fMRINames="${4}" 
	local ConcatName="${5}" 
	local HighPass="${6}"
```

Reply:
Ok